modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/wal/reader/StandaloneWalRecordsIteratorTest.java

```diff
@@ -273,9 +408,12 @@ private WALIterator createWalIterator(String walDir, WALPointer lowBound, WALPoi
         if (lowBound != null)
             params.from(lowBound);
 
-        if (lowBound != null)
+        if (highBound != null)
```

Comment:
Let's add a test for `highBound`?

Reply:
Actually, test for `highBound` exists.
Take a look at `testStrictBounds()`.
This change required, because, in my new tests only `lowBound` is set.